CMake/SlicerCPack.cmake

```diff
@@ -354,7 +354,9 @@ endif()
 # -------------------------------------------------------------------------
 if(CPACK_GENERATOR STREQUAL "NSIS")
 
-  set(Slicer_CPACK_NSIS_INSTALL_SUBDIRECTORY "")
+  if(NOT DEFINED Slicer_CPACK_NSIS_INSTALL_SUBDIRECTORY)
+    set(Slicer_CPACK_NSIS_INSTALL_SUBDIRECTORY "")
```

Comment:
Is there a difference in behavior if `Slicer_CPACK_NSIS_INSTALL_SUBDIRECTORY` is undefined vs. empty?

If yes, then we should described that in a comment.

If not (in most cases this is so, because the string value of an undefined CMake variable is `""`) then it would be simpler to just remove this line and add a comment that mentioning that `Slicer_CPACK_NSIS_INSTALL_SUBDIRECTORY` may be set externally.

Reply:
Being explicit makes sense as it is consistent with the `slicer_cpack_set` reporting an error if the variable is not defined.

Reply:
That's good it just need to be described in a comment. Otherwise this line may get removed during future refactoring.

Reply:
See suggestion above